src/components/Account.tsx

```diff
@@ -21,25 +21,27 @@ import useBalance from '../hooks/useBalance';
 import AccountDisplay from './AccountDisplay';
 
 interface Props {
-  value: string;
   accountName?: string;
-  className?: string;
   chain?: string | undefined;
+  className?: string;
+  hideAddress?: boolean;
   isDerived?: boolean;
   onClick?: () => void;
+  value: string;
 }
 
-const Account = ({ accountName, value, chain, isDerived = false }: Props) => {
+const Account = ({ accountName, value, chain, hideAddress = false, isDerived = false }: Props) => {
   const { api, address } = useApiBalance(value, chain, isDerived);
 
   const state = useBalance(api, address, true);
 
   return (
     <AccountDisplay
-      accountName={accountName}
+      friendlyName={accountName}
```

Comment:
sincerely i don't see the benefit of this replace:  if the value comes from an `accountName` why replacing the prop and diverge the name?

Reply:
It's because `AccountDisplay` is used in different context than `Account`. In this other context (Receiver Account) we can't really associated an "accountName` with the account, cause it's not coming from keystore/extension, but we still want to provide a name that is friendlier to the user.

If you insist I'm fine renaming it back to `accountName`, but IMHO it was misleading in context where `accountName` is just made up (like short address for instance).

Reply:
that's fine, leave as `friendlyName `